src/app/src/test/kotlin/org/imperial/mrc/hint/integration/BaselineTests.kt

```diff
@@ -50,6 +50,20 @@ class BaselineTests : SecureIntegrationTests() {
         }
     }
 
+    @ParameterizedTest
+    @EnumSource(IsAuthorized::class)
+    fun `can upload zip file as pjnz`(isAuthorized: IsAuthorized) {
+        val postEntity = getTestEntity("Botswana2018.zip")
```

Comment:
At the moment this `.zip` file is just a pjnz with a `.zip` extension, rather than a real zip of multiple pjnzs. Once hintr can support the latter should switch this test over to using a real example.

Reply:
Want to make a ticket to do this so we don't forget after this is merged?

Reply:
https://vimc.myjetbrains.com/youtrack/issue/mrc-699